CHANGELOG.md

```
@@ -24,6 +24,32 @@ Full Changelog: [https://github.com/netgrif/components/commits/v6.5.0](https://g
+- [NAE-2290] Couldn't find referenced task in a taskref incoming from event outcome
+- [NAE-2394] Fields looks editable when switching tabs
+
+### Added
+
+- [NAE-2100] Case view export button as NAE feature
+- [NAE-2393] Validation property on set data event
+- [NAE-2401] Timestamp of case dataSet change
+
+## [6.4.1](https://github.com/netgrif/components/releases/tag/v6.4.1) (2025-03-19)
+
+### Fixed
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)